klog.go

```diff
@@ -1265,6 +1302,33 @@ func Infof(format string, args ...interface{}) {
 	logging.printf(infoLog, logging.logr, format, args...)
 }
 
+// InfoS structured logs to the INFO log.
+// The msg argument used to add constant description to the log line.
+// The key/value pairs would be join by "=" ; a newline is always appended.
+//
+// Basic examples:
+// >> klog.InfoS("Pod status updated", "pod", "kubedns", "status", "ready")
+// output:
+// >> I1025 00:15:15.525108       1 controller_utils.go:116] "Pod status updated" pod="kubedns" status="ready"
+func InfoS(msg string, keysAndValues ...interface{}) {
+	logging.printS(nil, logging.logr, msg, keysAndValues...)
+}
+
+func kvListFormat(b *bytes.Buffer, keysAndValues ...interface{}) {
```

Comment:
Move this function after printS call? It would be closer to where it's used

Reply:
done